### PR TITLE
Fixed COLLADASW::Asset

### PR DESCRIPTION
**Pull request by jerome-jacob**
Restored original implementation + allow to write extra element in a derived class

### CODE REVIEW

**Comment by RemiArnaud** [comment]
@glaforte confirmed this fixes the asset problem recently introduced in max exporter.
Waiting for confirmation Maya exporter is fine.

**Comment by RemiArnaud** [comment]
tested on maya

File: COLLADAStreamWriter/src/COLLADASWAsset.cpp [hunk 1]
```diff
@@ -147,6 +119,10 @@ namespace COLLADASW
         default:
             mSW->appendTextElement ( CSWC::CSW_ELEMENT_UP_AXIS, CSWC::CSW_TEXT_Z_UP );
         }
+
+		addExtra();
```
Comment (mbarnes-sb) [inline COLLADAStreamWriter/src/COLLADASWAsset.cpp]: @RemiArnaud  As we discussed, It's invalid for <asset> to have <extra> element in 1.4.1. This change should be reverted and redesigned.

**Comment by mycheckgithub** [comment]
#414

**Pull request merged**